### PR TITLE
Write api filter book follow category_id and language

### PR DESCRIPTION
**Pull request by mvn-annguyen-dn**
## Description
Write api filter book follow category_id and language and filter book api
## Reference Requirements
- Product Backlog ID: 
- Task ID: 

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
LGTM

File: docs/api-references/books.md [hunk 1]
```diff
@@ -328,3 +328,207 @@ Get book reviews
     "total": 2
 }
 ```
+### `GET` Filter book with keyword.
+#### If exist category_id
+```
+    .../api/books?category=1
```
Comment (at-nhuibo) [inline docs/api-references/books.md]: Cái API này chính là cái api get list book á, dùng chung 1 cái thôi. Ko cần define ra từng loại.
Path thế này, gộp lại luôn.
```
/api/books?search=a&category=1& language=en
```

Thêm mấy cái params này vào **Request contents** á,
| Key | Requried | Value/Note |
|---|---|---|
|...|Optional|List hết 3 key|

**Review by ghost — CHANGES_REQUESTED** [review]
Check lại giúp a với nha!

**Review by ghost — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**